src/styles/blocks/header.scss

```diff
@@ -0,0 +1,50 @@
+html,
+body {
+  height: 100%;
+  margin: 0;
+}
+
+body {
+  background-image: url(../../images/header_main_image.png);
+  background-size: cover;
+  background-position: center;
+}
+
+.header {
+  width: 100%;
+  height: 100vh;
+  position: relative;
+  z-index: 1;
```

Comment:
Add the stylization I talked about the background-image here:

![image](https://github.com/user-attachments/assets/0fc4f2b0-4e87-4073-8ddb-45f115de8392)